main.py

```diff
@@ -10,9 +10,9 @@
 Pycon_year = "2016"
 Pycon_url = "https://tw.pycon.org"
 Written_url = []
+Visited_page = []
```

Comment:
It is a better habit for naming variable following PEP 8 coding style convention. For example, [the constant](https://www.python.org/dev/peps/pep-0008/#constants) variable should be written in all capital letters with underscores separating words, i.e. `PYCON_YEAR`, `PYCON_URL`, `WRITTEN_URL`... and [the variable](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) should be written in lowercase, with words separated by underscores